annoFuse/0.1.8/dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rocker/tidyverse:3.5.1
+MAINTAINER gaonkark@chop.email.edu
```

Comment:
MAINTAINER is deprecated: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
good to know! I can edit that if required, let me know.Thanks!